easysocial/easysocial/groupinvite.php

```diff
@@ -130,6 +141,56 @@ public function delete()
 		$this->plugin->setResponse($res);
 	}
 
+	/**
+	 * Method description Method used to leave group or delete group.
+	 *
+	 * @return  mixed
```

Comment:
@Pratikshakhandagale  - This function is not returning anything. So, either update the comments of this function of make it return something as per comments.